src/ui/component/fileTags/view.jsx

```diff
@@ -1,19 +1,25 @@
 // @flow
 import * as React from 'react';
+import classnames from 'classnames';
 import Button from 'component/button';
 
-const MAX_TAGS = 4;
+const SLIM_TAGS = 2;
+const NORMAL_TAGS = 4;
+const LARGE_TAGS = 10;
 
 type Props = {
   tags: Array<string>,
   followedTags: Array<Tag>,
+  large: boolean,
+  slim: boolean,
```

Comment:
slightly worse pattern, because passing true for both large and slim should not be possible and would have unexpected behavior

Reply:
Changed to `type={"small", "large"}`